assets/js/combine.js

```
@@ -5,11 +5,11 @@ const indexPath = 'index.html';
 let indexHtml = fs.readFileSync(indexPath, 'utf8');
 
 // Combine CSS
-const cssFiles = ['assets/css/site.css', 'public/assets/css/style.css'];
+const cssFiles = ['assets/css/main.css'];
 let cssCombined = cssFiles.map(file => fs.readFileSync(file, 'utf8')).join('\n');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'cssCombined' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)